samples/soundtest.lobster

```diff
@@ -0,0 +1,173 @@
+// Extension of shooter tutorial tut7 here with background music, enemy sounds and "hot" pause
```

Comment:
put this in the same dir as the shooter tutorial.. call it `tut_sound.lobster` maybe. At the end of the shooter tutorial doc (just before all the coroutine stuff) make a paragraph saying.. want to learn how to add sound to this code? have a look at this file, and the functions starting with `play_` or `sound_`.